packages/editor/src/systems/EditorFlyControlSystem.ts

```diff
@@ -55,11 +51,11 @@ const onSecondaryClick = () => {
 const onSecondaryReleased = () => {
   const camera = getComponent(Engine.instance.cameraEntity, CameraComponent)
   if (hasComponent(Engine.instance.cameraEntity, FlyControlComponent)) {
-    const distance = camera.position.distanceTo(editorCameraCenter)
-    editorCameraCenter.addVectors(
-      camera.position,
-      tempVec3.set(0, 0, -distance).applyMatrix3(normalMatrix.getNormalMatrix(camera.matrix))
-    )
+    // const distance = camera.position.distanceTo(editorCameraCenter)
```

Comment:
is this intention or left over from testing? can this be re-enabled?

Reply:
I reimplemented this functionality to be fully based around the transform component rather than the threejs camera object. Just forgot to remove these old comments. Might be better to place the reimplementation here rather than doing it in the camera fly system.